### PR TITLE
Update Group1_Project_Part3.qmd

### PR DESCRIPTION
**Pull request by tdchoi7**

### CODE REVIEW

**Review by tdchoi7 — COMMENTED** [review]
No need for change

**Review by zxhchgl — COMMENTED** [review]
1

File: Group1_Project_Part3.qmd
Comment (zxhchgl) [inline Group1_Project_Part3.qmd:1]: 1

**Review by zxhchgl — APPROVED** [review]

**Review by zxhchgl — CHANGES_REQUESTED** [review]
1

File: Group1_Project_Part3.qmd
Comment (tdchoi7) [reply Group1_Project_Part3.qmd:1]: 1

**Pull request closed without merging**